internal/compiler-bridge/src/main/scala/xsbt/Dependency.scala

```diff
@@ -126,10 +126,13 @@ final class Dependency(val global: CallbackGlobal) extends LocateClassFile with
             // The dependency comes from a class file
             binaryDependency(pf.file, binaryClassName)
           case _ =>
-            reporter.error(
-              NoPosition,
-              s"Internal error: ${binaryClassName} comes from unknown origin ${at}"
-            )
+          // On Scala 2.10 you get Internal error: <none> comes from unknown origin null
+          // if you uncomment the following:
```

Comment:
There's already an issue for this - https://github.com/sbt/zinc/issues/604

Reply:
@eed3si9n @jvican We did not apply this patch in our hydra-bridge and we started seeing these errors in Java 11. Perhaps this means that there is a bug in incremental compilation? Uncomment and check compiling Akka on Java 11, you'll see many of these even with Scala 2.12.8. /cc @johanandren

Reply:
Yeah, those are well known issues, happen with symbols coming from jrt file systems. In Java 11s case, it comes from symbols in the default java library. Not an issue so long as people don’t use JRT file systems in their classpath.

Reply:
At any rate, the comment is wrong. It is the case on any Scala version, and for other symbols than `<none>`. Not sure if you want to eventually fix it, just pointing out that there *might* be a bug lurking here.